hbase-client/src/main/java/org/apache/hadoop/hbase/ipc/NettyRpcConnection.java

```diff
@@ -216,24 +220,33 @@ public void operationComplete(Future<Boolean> future) throws Exception {
     });
   }
 
-  private void connect() {
+  private void connect() throws UnknownHostException {
     if (LOG.isDebugEnabled()) {
-      LOG.debug("Connecting to " + remoteId.address);
+      LOG.debug("Connecting to " + remoteId.getAddress());
```

Comment:
This is the netty version of the bug fix here: https://github.com/apache/hbase/pull/2671/files#diff-1a7ec27a8107293b6c87132823c262fc250570687a40f45646c43ae46dc6b04eR255